CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 23.3.2 [#700](https://github.com/openfisca/openfisca-core/pull/700)
+
+* Send the web API user's IP address to the tracker even when it goes through a proxy.
```

Comment:
I feel this changelog reads very ominous and gives a wrong idea about how tracking is done.
Suggestion: `Fix API source IP detection through proxies`.